Source/Base/FieldContainer/Base/OSGFieldContainer.cpp

```diff
@@ -268,6 +268,83 @@ bool FieldContainer::unlinkChild(FieldContainer * const pChild,
     return false;
 }
 
+void FieldContainer::verifyThreadSafety(ConstFieldMaskArg whichField)
+{
+    std::string reason("");
+
+    // It should not be possible to not have _pContainerChanges at this
```

Comment:
nitpick: The double negative here tripped me up a bit.

Reply:
Done